packages/manager/src/factories/dashboards.ts

```diff
@@ -62,25 +63,41 @@ export const dashboardMetricFactory =
       (i) => scrape_interval[i % scrape_interval.length]
     ),
     unit: 'defaultUnit',
-  });
+  }
+);
 
-export const cloudPulseMetricsResponseDataFactory =
-  Factory.Sync.makeFactory<CloudPulseMetricsResponseData>({
+export const cloudPulseMetricsResponseDataFactory = Factory.Sync.makeFactory<CloudPulseMetricsResponseData>(
+  {
     result: [
       {
         metric: {},
         values: [],
       },
     ],
     result_type: 'matrix',
-  });
+  }
+);
 
-export const cloudPulseMetricsResponseFactory =
-  Factory.Sync.makeFactory<CloudPulseMetricsResponse>({
+export const cloudPulseMetricsResponseFactory = Factory.Sync.makeFactory<CloudPulseMetricsResponse>(
+  {
     data: cloudPulseMetricsResponseDataFactory.build(),
     isPartial: false,
     stats: {
       series_fetched: 2,
     },
     status: 'success',
-  });
+  }
+);
+
+export const userPreferencesFactory = Factory.Sync.makeFactory<UserPreferences>(
+  {
+    aclpPreference: {
+      dashboardId: 'default-id',
+      engine: 'default-engine',
+      region: 'us-ord',
+      resources: ['1'],
+      role: 'default-role',
```

Comment:
Should we replace this "role" with "node_type" here as well?

Reply:
@venkymano-akamai / @nikhagra-akamai  fyi

Reply:
I am setting the correct node type value at the test case level, as this is the default value required for proper functionality.